esp/esp/program/modules/handlers/onsiteprintschedules.py

```diff
@@ -56,7 +56,7 @@ def module_properties(cls):
             }
 
     @main_call
-#    @needs_onsite
+    @needs_onsite
```

Comment:
Oh, weird, looks like we have two copies of the print schedules script floating around -- the other one was fixed in 4368beb7b.  In any case, we can clean that up later.